packages/selenium/test/RunPartial/UseLegacyTests.cs

```diff
@@ -0,0 +1,51 @@
+using System.Linq;
+using Deque.AxeCore.Selenium;
+using NUnit.Framework;
+
+namespace Deque.AxeCore.Selenium.Test.RunPartial
+{
+    public class UseLegacyTests : TestBase
+    {
+        [Test]
+        public void ShouldRunLegacyAxe()
+        {
+            GoToFixture("index.html");
+
+            var results = new AxeBuilder(driver, CustomSource($"{axeSource}{axeRunPartialThrows}"))
+                .UseLegacyMode(true)
+                .Analyze();
+
+            Assert.IsNotNull(results);
+            Assert.IsTrue(results.Passes.Any());
+        }
+
+        [Test]
+        public void ShouldNotTestCrossOriginFrames()
+        {
+            GoToUrl("http://localhost:8080/cross-origin.html");
+
+            var results = new AxeBuilder(driver, CustomSource($"{axeSource}{axeRunPartialThrows}"))
+                .UseLegacyMode(true)
+                .WithRules("frame-tested")
+                .Analyze();
+
+            var frameTested = results.Incomplete.FirstOrDefault(x => x.Id == "frame-tested");
+            Assert.IsNotNull(frameTested);
+        }
+
+        [Test]
+        public void ShouldBeDisabledAgain()
+        {
+            GoToUrl("http://localhost:8080/cross-origin.html");
```

Comment:
I would expect this test to fail in a false positive as-written because `cross-origin.html` includes an iframe element that points to `http://127.0.0.1`, but the test and the `serveFixtures` run script are written assuming that the fixture server is being hosted on port `8080` (not `80`) - the `http://127.0.0.1` iframe will 404 and that should result in a frame-tested failure occurring even in `runPartial` mode.

Reply:
I investigated this a bit more because it confused me that this test was passing. It turns out that it's passing because invoking `EvaluateAsync` on a frame pointing to a non-responsive navigation target isn't an error like I'd have expected - it just runs a scan against the browser's built-in "127.0.0.1 refused to connect." error page, which is enough to make this test valid at least.